docs/en/cloud/security/ip-public-api.md

```diff
@@ -0,0 +1,49 @@
+---
+slug: /en/manage/security/ip-public-api
+sidebar_label: Public IP API
+title: Public IP API
+---
+
+## ClickHouse Cloud public API
+
+The ClickHouse Cloud public API endpoint is available on [`https://api.clickhouse.cloud/static-ips.json`](https://api.clickhouse.cloud/static-ips.json).
+
+This API provides public IPs and endpoints for ClickHouse services, such as ingress/egress IPs and AWS S3 endpoints per region and cloud.
+
+## Integrations could require egress access
+
+If you are using integration like the MySQL or PostgreSQL Engine, it is possible that you need to authorize ClickHouse Cloud to access to your instances.
+
+You could use this API of public IPs in `firewalls` or `Authorized networks` in GCP or in `Security Groups` for Azure, AWS or in any other infrastructure egress management system you are using.
+
+For example, to allow access from a ClickHouse Service hosted on AWS in the region ap-south-1, you can add the `egress_ips` addresses for that region:
+
+```
+❯ curl -s https://api.clickhouse.cloud/static-ips.json | jq '.'
+{
+  "aws": [
+    {
+      "cell": "cell0",
+      "egress_ips": [
+        "3.110.39.68",
+        "15.206.7.77",
+        "3.6.83.17"
+      ],
+      "ingress_ips": [
+        "15.206.78.111",
+        "3.6.185.108",
+        "43.204.6.248"
+      ],
+      "region": "ap-south-1",
+      "s3_endpoint": "vpce-0a975c9130d07276d"
+    },
+...
+```
+
+For example a ClickHouse cloud service running in `us-east-2` using the integration to connect to an RDS in AWS, should have the following Inbound security group rules:
+
+<img src={require("@site/docs/en/_snippets/images/aws-rds-mysql.png").default} style={{width:'70%', padding:'20px'}} alt="AWS Security group rules"/>
```

Comment:
```suggestion
![AWS Security group rules](@site/docs/en/_snippets/images/aws-rds-mysql.png) 
```

I can't read the image with those settings.

Reply:
Interesting. It works here locally:

<img width="864" alt="image" src="https://user-images.githubusercontent.com/19713226/229879584-300d5490-2b07-42e2-b15e-8ef8a987baf1.png">